src/Gateio/Futures/API/OrderBook.jl

```diff
@@ -0,0 +1,104 @@
+module OrderBook
+
+export OrderBookQuery,
+    OrderBookData,
+    order_book
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Gateio
+using CryptoAPIs: Maybe, APIsRequest
+
+Base.@kwdef struct OrderBookQuery <: GateioPublicQuery
+    contract::String
+    interval::Maybe{String} = nothing
+    limit::Maybe{Int64} = nothing
+    with_id::Maybe{Bool} = nothing
+end
+
+struct Order <: GateioData
+    p::String
+    s::Int64
+end
+
+struct OrderBookData <: GateioData
+    id::Maybe{Int64}
+    current::Float64
+    update::Float64
```

Comment:
Looks like it should be NanoDates

Reply:
No, it seems to be a float response for some reason
<img width="187" alt="image" src="https://github.com/bhftbootcamp/CryptoAPIs.jl/assets/84569060/39b14c35-eaee-4976-9a63-d146a0e3fd06">

Reply:
If you multiply the return values by 100, then this will be considered a completely valid NanoDate, so I see no point in keeping them as a Float
![image](https://github.com/bhftbootcamp/CryptoAPIs.jl/assets/153018291/628275e3-faa1-49d8-870a-dae7211ed27b)